homeassistant/components/manual/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "domain": "manual",
-  "name": "Manual",
-  "documentation": "https://www.home-assistant.io/integrations/manual",
+  "name": "Manual Alarm Control Panel",
+  "documentation": "https://www.home-assistant.io/integrations/alarm_control_panel.manual",
```

Comment:
This change is incorrect.
Integration should be listed by their domain key in our documentation.

Reply:
Those haven't been migrated yet, most has been migrated.

Reply:
Could you please elaborate on what you mean by "migrated"? What is the direction things should move towards?

Reply:
All documentation will be listed under the domain key. As has been done for almost every integration.